spec/component/models/activity_insight_oa_file_spec.rb

```diff
@@ -60,25 +60,34 @@
     }
     let!(:pub2) { create(:publication,
                          title: 'pub2',
-                         publication_type: 'Trade Journal Article')
+                         publication_type: 'Trade Journal Article',
+                         open_access_status: 'gold')
```

Comment:
I believe we are testing the exclusion of 'Trade Journal Article' from the `ready_for_download` filter with this record, so we probably want to leave the open access status as nil or 'green' (or anything that would be included).